code/modules/tooltip/tooltip.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tooltip</title>
+	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<style type="text/css">
+		body, html {
+			margin: 0;
+			padding: 0;
+			overflow: hidden;
+		}
+
+		.wrap {
+			position: absolute;
+			top: 0;
+			left: 0;
+			max-width: 298px;
+			border: 2px solid #1B2967;
+		}
+
+		.content {
+			font: bold 12px Arial, 'Helvetica Neue', Helvetica, sans-serif;
+			color: #ffffff;
+			padding: 8px;
+			border: 2px solid #0033CC;
+			background: #005CB8;
+		}
+
+		h1 {
+			margin: -5px 0 2px 0;
+			font-size: 1.2em;
+			line-height: 1.4;
+		}
+
+		p {
+			margin: 0;
+			line-height: 1.2;
+		}
+
+		/* Custom Themes */
+		.blob .wrap {border-color: #009900;}
+		.blob .content {border-color: #66FF00; background-color: #475E13;}
+
+		.wraith .wrap {border-color: #492136;}
+		.wraith .content {border-color: #331726; background-color: #471962;}
+
+		.pod .wrap {border-color: #052401;}
+		.pod .content {border-color: #326D29; background-color: #569F4B;}
+
+		.colo-pod .wrap {border-color: #256fb9;}
+		.colo-pod .content {border-color: #000000; background-color: #000000;}
+		
+		
+		/* TG: Themes */
+		/* ScreenUI */
+		.midnight .wrap {border-color: #2B2B33;}
+		.midnight .content {color: #6087A0; border-color: #2B2B33; background-color: #36363C;}
+		
+		.plasmafire .wrap {border-color: #21213D;}
+		.plasmafire .content {color: #FFA800 ; border-color: #21213D; background-color:#1D1D36;}
+
+		.retro .wrap {border-color: #005E00;}
+		.retro .content {color: #003366; border-color: #005E00; background-color: #00BD00;}
+		
+		
+		
+	</style>
+</head>
+<body>
+	<div id="wrap" class="wrap">
+		<div id="content" class="content"></div>
+	</div>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
```

Comment:
I think we already send minified jquery somewhere might use that instead, but it's pretty irrelevant anyway.